### ISSUE TITLE
private AutoProperties to public fields

### ISSUE THREAD

**Issue by oliver-h** [nhabuiduc/TypescriptSyntaxPaste#5]
In Version 1.1., installed via VS "Extensions and Updates", private AutoProperties are translated to public fields.

input:

```
private static int PrivateStaticGetter { get; set; }
private int PrivateGetter { get; set; }
```

output:

```
public PrivateStaticGetter: number;
public PrivateGetter: number;
```

**Comment by oliver-h** [comment]
Also, I noticed now, that public static AutoProperties lose the "static".

**Comment by nhabuiduc** [comment]
Fixed for that problem in commit 664b260e055c11d94f296aa993aaf7e99b921d1a

**Comment by oliver-h** [comment]
Thank you!
